NPS.ID.PublicApi.DotNet.Client/Connection/WebSocketConnectorFactory.cs

```diff
@@ -0,0 +1,25 @@
+using Microsoft.Extensions.Logging;
+using NPS.ID.PublicApi.DotNet.Client.Connection.Options;
+using NPS.ID.PublicApi.DotNet.Client.Security;
+using NPS.ID.PublicApi.DotNet.Client.Security.Options;
+
+namespace NPS.ID.PublicApi.DotNet.Client.Connection;
+
+public class WebSocketConnectorFactory : IWebSocketConnectorFactory
```

Comment:
Do we really need this abstraction? I think the code can be inlined to where it's used

Reply:
I mean, we have the same implementation in ID3 Stomp Client so I would prefer to stay consistent. That's why i decided to implement it the same way.

Reply:
Removed not needed abstractions from WebSocketConnectorFactory and some others